.github/scripts/setup_spark_remote.sh

```diff
@@ -34,4 +34,4 @@ if [ $? -ne 0 ]; then
     #sleep time to start the server
     sleep 2m
     echo "Server Already Running"
-fi
+fi
```

Comment:
remove this space so there is no change here.

Reply:
done